src/main/java/com/aws/greengrass/clientdevices/auth/certificate/listeners/ConfiguredCertificateAuthorityEventListener.java

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Could we place the listeners on a package inside the domain they belong to?

Reply:
Happy to make structural changes. Just kinda threw this in a listeners package. But this is still certificate related. Are you suggesting we break CA registration into a separate domain?

Reply:
This was my bad. My brain is fried